_observing-your-data/prometheusmetrics.md

```diff
@@ -0,0 +1,116 @@
+
+![Prometheus example 1]({{site.url}}{{site.baseurl}}/images/metrics/metrics4.png)
+
+Upon selection of a metric, the **Metrics explorer** will create a visualization for the selected metric to be displayed, as seen in the following image.
+
+![Prometheus example 2]({{site.url}}{{site.baseurl}}/images/metrics/metrics5.png)
+
+### Configuring a Prometheus data source connection to OpenSearch
+
+In order to configure a connection to Prometheus for metrics in OpenSearch Dashboards, you will need to create a file on your OpenSearch nodes named `datasources.json` containing the Prometheus data source settings. The following examples demonstrate the various Prometheus data source configurations using different authentication methods:
+
+- No authentication:
+
+   ```json
+   [{
+       "name" : "my_prometheus",
+       "connector": "prometheus",
+       "properties" : {
+           "prometheus.uri" : "http://localhost:9090"
+       }
+   }]
+   ```
+
+- Basic authentication:
+
+   ```json
+   [{
+       "name" : "my_prometheus",
+       "connector": "prometheus",
+       "properties" : {
+           "prometheus.uri" : "http://localhost:9090",
+           "prometheus.auth.type" : "basicauth",
+           "prometheus.auth.username" : "admin",
+           "prometheus.auth.password" : "admin"
+      }
+   }]
+   ```
+
+- AWSSigV4 authentication:
+
+   ```json
+   [{
+       "name" : "my_prometheus",
+       "connector": "prometheus",
+       "properties" : {
+           "prometheus.uri" : "http://localhost:8080",
+           "prometheus.auth.type" : "awssigv4",
+           "prometheus.auth.region" : "us-east-1",
+           "prometheus.auth.access_key" : "{{accessKey}}"
+           "prometheus.auth.secret_key" : "{{secretKey}}"
+       }
+   }]
+   ```
+
+After entering your Prometheus configuration into the `datasources.json` file, run the following command to load the configuration into the OpenSearch keystore. This is securely stored as is contains credential information.
+
+```
+bin/opensearch-keystore add-file plugins.query.federation.datasources.config datasources.json
+```
+
+If you are updating the keystore during runtime, refresh the keystore with the following API command:
+
+```bash
+POST /_nodes/reload_secure_settings
+{
+  "secure_settings_password":""
+  
+}
+```
+{% include copy-curl.html %}
+
+Metrics collected from Prometheus will now be displayed in the **Observability** application under **Metrics analytics**. For further information, see the [Prometheus Connector](https://github.com/opensearch-project/sql/blob/main/docs/user/ppl/admin/prometheus_connector.rst) GitHub page.
+
+### Prometheus PPL queries
+
+Below is an example of a [Piped Processing Language]({{site.url}}{{site.baseurl}}/search-plugins/sql/ppl/index) (PPL) query using a Prometheus connection in OpenSearch. This example demonstrates a metric selecting query with specific dimensions:
```

Comment:
Do we need to provide a brief intro sentence here on what a Prometheus PPL query is or why you would need it?

Reply:
PPL querys are not unique to Prometheus, and I also linked to relevant documentation on PPL with OpenSearch

Reply:
Right, but why is it relevant to point out here? There must be something unique about it?

Reply:
it's relevant because being able to use PPL queries against Prometheus metrics is a new ability that you used to not be able to do. So this lets the user know that it is possable to use PPL with the Prometheus connection.